src/app/components.tsx

```diff
@@ -0,0 +1,62 @@
+import Image from "next/image";
+import logo from "../assets/logo.png";
+import styles from "./page.module.scss";
+
+export { ModeToggleButton, Logo, Searchbar, KeywordRadioBtn };
```

Comment:
Move these components into a `/components folder and have each component in its own folder.

Reply:
ok done